Makefile

```diff
@@ -135,6 +135,23 @@ test-unit-js:
 		yarn test \
 	'
 
+# temp target for quick building and testing
+# can remove once we have a test harness around the Docker image
+.PHONY: go-build-git-initializer
+go-build-git-initializer:
+	@cd v2/git-initializer && \
+		go build \
+			-o ../../bin/git-initializer \
+			-ldflags "-w \
+				-X github.com/brigadecore/brigade/v2/internal/version.version=${VERSION} \
+				-X github.com/brigadecore/brigade/v2/internal/version.commit=${GIT_VERSION}" \
+			.
+
+.PHONY: test-git-initializer
+test-git-initializer: go-build-git-initializer
+	@cd v2/git-initializer && \
+		./test.sh
+
```

Comment:
See [this comment]( https://github.com/brigadecore/brigade/pull/1229/commits/b591990ca5dff04b1bfbd2cbee45cb775954da54#r557473549)

I'd prefer not introducing a suite of bash-based tests if we can avoid it.

Reply:
Removed.